web/public/locales/ro.json

```diff
@@ -1,6 +1,6 @@
 {
     "panel-initial-text": {
-        "thisproject": "Acest proiect este",
+        "thisproject": "Acest app este",
```

Comment:
I did for most of them, but I must admit I figured someone would adjust the translation quite rapidly if a mistake was made.
The main rationale for doing this is to enable us to be perceived as a serious app instead of a side project.